src/explore-education-statistics-common/src/components/Accordion.tsx

```diff
@@ -25,9 +24,31 @@ export interface AccordionProps {
 const Accordion = ({ children, id, onToggleAll }: AccordionProps) => {
   const ref = useRef<HTMLDivElement>(null);
 
-  const [hashId, setHashId] = useState('');
   const [openSections, updateOpenSections] = useImmer<boolean[]>([]);
 
+  const sections = React.Children.toArray(children).filter(child =>
+    isComponentType(child, AccordionSection),
+  ) as ReactComponentElement<typeof AccordionSection>[];
+
+  const getSectionIds = (
+    section: ReactComponentElement<typeof AccordionSection>,
```

Comment:
Only need the props here, so would just pass `section.props` through instead and type the parameter as `AccordionSectionProps` (it's a bit of a mouthful as `ReactComponentElement<typeof AccordionSection>`